packages/region-helper/test/index.test.ts

```diff
@@ -39,6 +39,10 @@ describe('getRegion', () => {
     expect(getRegion(1)).toBe(EU_CODE)
   })
 
+  it('should return `cn` region when preferred region is cn', () => {
+    expect(getRegion(1, 'cn')).toBe(CN_CODE)
+  })
```

Comment:
It would be interesting to have an extra test for EU:

````js
it('should return `EU` region when preferred region is cn', () => {
    expect(getRegion(1, 'eu')).toBe(EU_CODE)
  })

```